lmfdb/characters/main.py

```diff
@@ -156,6 +164,24 @@ def common_parse(info, query):
     parse_ints(info, query, "modulus", name="modulus")
     parse_ints(info, query, "conductor", name="conductor")
     parse_ints(info, query, "order", name="order")
+    if 'inducing' in info:
+        try:
+            validate_label(info['inducing'])
+            parts_of_label = info['inducing'].split(".")
+            if len(parts_of_label) != 2 or not str.isalpha(parts_of_label[1]):
+                raise ValueError("Invalid character orbit label format, expected N.a")
+            primitive_modulus = int(parts_of_label[0])
+            primitive_orbit = class_to_int(parts_of_label[1])+1
```

Comment:
Is it worth supporting a Conrey label for an actual character, instead of just an orbit?

Reply:
Done, seems fast enough.